muesli/web/templates/lecture/view.pt

```diff
@@ -48,9 +48,12 @@
           <a tal:attributes="href request.route_path('tutorial_view', tutorial_ids=tutorial.id)">Details</a>
         </td>
         <td>
-          <a tal:condition='tutorial.student_count &lt; tutorial.max_students '
+          <a tal:condition='(tutorial.student_count &lt; tutorial.max_students) and tutorial.id != subscribed_tutorial_id'
              tal:attributes="href request.route_path('tutorial_subscribe', tutorial_id=tutorial.id)">
-             ${'Wechseln' if subscribed else 'Beitreten'}
+             ${'Wechseln' if subscribed_to_any_tutorial else 'Beitreten'}
+          </a>
+          <a tal:condition='tutorial.id == subscribed_tutorial_id' tal:attributes="href request.route_path('tutorial_unsubscribe', tutorial_id=tutorial.id)">
```

Comment:
Can you put this in the above query as well? You could use something like
`${'Beitreten' if not subscribed_tutorial else if tutorial == subscribed_tutorial 'Austreten' else 'Wechseln'}` and use the same kind of logic to select the href.

Reply:
I've adjusted the logic to (hopefully) work now. I still have two links (with conditions that should be mutually exclusive), because I believe that having one link would make the logic unnecessarily hard to read.